Code/GraphMol/Substruct/catch_tests.cpp

```diff
@@ -127,4 +167,72 @@ TEST_CASE("providing a final match function", "[substruct]") {
     ps.extraFinalCheck = &bigger;
     CHECK(SubstructMatch(*mol1, *mol2, ps).size() == 1);
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE("Enhanced stereochemistry", "[substruct,StereoGroup]") {
+  // Chirality specifications.
+  // 1. An achiral molecule: CC(O)C(CC)F means unknown/all stereoisomers
+  // 2. A chiral molecule: C[C@H](O)[C@H](CC)F means 1 stereoisomer
+  // 3. A chiral molecule with an AND specifier: C[C@H](O)[C@H](CC)F |a1:1,3|
+  // means both stereoisomers
+  // 4. A chiral molecule with an OR specifier: C[C@H](O)[C@H](CC)F |o1:1,3|
+  // means one of the two stereoisomers
+  auto mol_achiral = "CC(O)C(CC)F"_smiles;
+  auto mol_chiral = "C[C@H](O)[C@H](CC)F"_smiles;
```

Comment:
this set of tests should also include molecules that have one and both of the stereocenters flipped. Maybe:
```
auto mol_chiral_2flip = "C[C@@H](O)[C@@H](CC)F"_smiles;
auto mol_chiral_1flip = "C[C@H](O)[C@@H](CC)F"_smiles;
```
The behavior of these in the various test cases will also be informative

Reply:
Ah, I see you have these below already, but I think it's clearer to move them up here and use them throughout the tests.

Reply:
good call.